### PR TITLE
EC2 Instance Resources

### PR DESCRIPTION
**Pull request by Subhadeep1997**

### CODE REVIEW

**Comment by Subhadeep1997** [comment]
EC2 Instance Resources

**Comment by gavinyue** [comment]
Hey, 

Could you provide an example output from the ec2 instance provider?

Thanks,

**Comment by gavinyue** [comment]
Btw, you can update the test cased with AWS-Mock simply

**Comment by Subhadeep1997** [comment]
Screenshot 2020-06-30 at 11 41 03 PM

**Comment by Subhadeep1997** [comment]
Screenshot 2020-07-01 at 3 42 41 PM

**Comment by Subhadeep1997** [comment]
Screenshot 2020-07-01 at 4 10 55 PM

**Comment by Subhadeep1997** [comment]
Screenshot 2020-07-02 at 10 20 08 AM

**Comment by Subhadeep1997** [comment]
Screenshot 2020-07-02 at 10 44 43 AM

**Comment by Subhadeep1997** [comment]
Screenshot 2020-07-02 at 5 28 09 PM

File: aws/src/aws/helper/aws-clients-helper.ts [hunk 1]
```diff
@@ -2,13 +2,19 @@ import * as AWS from 'aws-sdk';
...
     let client;
     switch (serviceName) {
       case 'S3':
         client = new AWS.S3({
           region: region,
         });
         break;
+      case 'EC2':
+        client = new AWS.EC2({
+          apiVersion: '2016-11-15',
```
Comment (gavinyue) [inline aws/src/aws/helper/aws-clients-helper.ts:15]: Add the region. 

I will handle it.

**Pull request merged**